server/gokbg3/grails-app/services/org/gokb/FTUpdateService.groovy

```diff
@@ -437,14 +437,18 @@ class FTUpdateService {
         // title history for all title types
         if (kbc.title?.hasProperty("historyEvents")){
```

Comment:
`historyEvents` is not defined. Just removing the check would be fine.

Reply:
True. This is a relict, and non-functional as well.